data/ssystem_minor.ini

```diff
@@ -1799,7 +1992,10 @@ dust_brightnessfactor          = 1.5
 dust_lengthfactor              = 0.4
 dust_widthfactor               = 1.5
 model                          = 1682q1halley_MLfix.obj
-name                           = P/Halley (1982i=1986III)
+name                           = Comet Halley
+date_code                      = 1P/1982 U1
```

Comment:
This should now be iau_designation

Reply:
Well, according MPC the comet _1P/Halley_ has IAU designation _P/1682 Q1_, so, the designation _1P/1982 U1_ should be added as `date_code`

Reply:
Ah, OK. So my fix for all Halley entries in 1000_comets.ini is incomplete/wrong, I should replace iau_designation by date_code. But then this entry still needs iau_designation=1P/1682 Q1.